_posts/2017-12-19-master-thesis.md

```diff
@@ -0,0 +1,54 @@
+---
+layout: post
+title: "Meine Masterarbeit mit karriere.at"
+date: 2017-12-19 08:00:00
+author: stefank
+tags: [informationextraction, masterthesis, fhhagenberg, parttime]
```

Comment:
Hast du kontrolliert, ob es diese Tags schon in einer bestimmten Schreibweise gibt?
Wir sollten da wohl eine Übersichtsseite machen, damit man bestehende Tags sieht.